tests/sentry/api/path_params/resources/test_id_or_slug_path_params_mixin.py

```diff
@@ -0,0 +1,83 @@
+from typing import Any
+
+
+class APIIdOrSlugTestMixin:
+    slug_mappings: dict[str, Any]
+    reverse_slug_mappings: dict[str, Any]
+    incident: Any
+    code_mapping: Any
+    incident_activity: Any
```

Comment:
in a future PR we should try to narrow these type defs as we know more about what's going into them